.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "cortex-debug.armToolchainPath": "/usr/local/bin/",
```

Comment:
On my machine debugging works without these settings. Maybe to remove them and this way it'll become cross-platform?